fvm/src/syscalls/actor.rs

```diff
@@ -109,3 +112,19 @@ pub fn get_code_cid_for_type(
     let k = context.kernel.get_code_cid_for_type(typ)?;
     context.memory.write_cid(&k, obuf_off, obuf_len)
 }
+
+pub fn become_actor(
+    context: Context<'_, impl Kernel>,
+    code_cid_off: u32, // Cid
+) -> std::result::Result<Never, Abort> {
+    // TODO different exit code?
+    let cid = context
+        .memory
+        .read_cid(code_cid_off)
+        .map_err(|e| Abort::from_error(ExitCode::USR_ILLEGAL_ARGUMENT, e))?;
+    context.kernel.become_actor(&cid)?; // ExitCode::SYS_INVALID_RECEIVER
+    Err(Abort::Exit(
+        ExitCode::OK,
+        "became a new actor something something".to_string(),
+    ))
```

Comment:
Hm. So, the tricky part will be returning a value.

Reply:
We probably need a new variant in `Abort::` along the lines of `Abort::Return`. (and maybe we should rename `Abort` to `EarlyExit`, or something like that...

Reply:
I like `Terminate`